up42/workflow.py

```diff
@@ -327,7 +327,7 @@ def construct_parameters(
                 input_parameters[data_block_name]["limit"] = len(scene_ids)
                 input_parameters[data_block_name].pop("time")
             elif start_date is not None and end_date is not None:
-                datetime = f"{start_date}T00:00:00Z/{end_date}T00:00:00Z"
+                datetime = f"{start_date}T00:00:00Z/{end_date}T23:59:59Z"
                 input_parameters[data_block_name]["time"] = datetime
```

Comment:
I guess there should be a tests specifically for this no? I see you changed the params but not sure if you input the same the result would be different.

Reply:
Hm guess more of an api behaviour than something that we need to test in the SDK. Just tried in Postman and can confirm that `"2019-01-14T00:00:00Z/2019-01-15T23:59:59Z"` can find a scene on the 15th, but `"2019-01-14T00:00:00Z/2019-01-15T00:00:00Z"` does not. Will change the default dates in the postman examples.